Client/Frontend/Browser/Playlist/Controllers/PlaylistMoveFolderView.swift

```diff
@@ -0,0 +1,269 @@
+                    }
+                }
+                .listRowBackground(Color(.secondaryBraveGroupedBackground))
+                
+                let sectionTitle = selectedItems.count == 1 ? Strings.PlaylistFolders.playlistFolderSelectASingleFolderTitle : Strings.PlaylistFolders.playlistFolderSelectAFolderTitle
+                Section(header: Text(sectionTitle)
+                            .font(.footnote)
+                            .textCase(.none)
+                            .foregroundColor(Color(.secondaryBraveLabel))
+                            .multilineTextAlignment(.leading)) {
+                    
+                    // Show the "Saved" folder
+                    if PlaylistManager.shared.currentFolder?.uuid != savedFolder.first?.uuid {
+                        Button(action: {
+                            selectedFolder = savedFolder.first
+                            moveDisabled = false
+                        }) {
+                            PlaylistFolderView(iconColor: Color(.braveOrange),
+                                               titleColor: .white,
+                                               folder: savedFolder.first,
+                                               selectedFolder: $selectedFolder)
+                        }
+                    }
+                    
+                    // Show all folders except the current one
+                    ForEach((0..<folders.count), id: \.self) { index in
+                        if folders[index].uuid != PlaylistManager.shared.currentFolder?.uuid {
+                            Button(action: {
+                                selectedFolder = folders[index]
+                                moveDisabled = false
+                            }) {
+                                PlaylistFolderView(iconColor: Color(.braveOrange),
+                                                   titleColor: .white,
+                                                   folder: folders[index],
+                                                   selectedFolder: $selectedFolder)
+                            }
+                        }
+                    }
+                }
+                .listRowBackground(Color(.secondaryBraveGroupedBackground))
+            }
+            .listStyle(.insetGrouped)
+            .navigationTitle(Strings.PlaylistFolders.playlistFolderMoveFolderScreenTitle)
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationViewStyle(.stack)
+            .toolbar {
+                ToolbarItem(placement: .navigationBarLeading) {
+                    Button(Strings.cancelButtonTitle) { onCancelButtonPressed?() }
+                    .foregroundColor(.white)
+                }
+                
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Button(Strings.done) { onDoneButtonPressed?(selectedItems, selectedFolder) }
+                    .foregroundColor(moveDisabled ? .gray : .white)
```

Comment:
If iOS isn't changing the color based on the disabled status automatically then: `.gray` → `Color(.braveDisabled)` and if thats also the case then the same foreground color check is missing in `PlaylistEditFolderView`